notes-service/pom.xml

```diff
@@ -11,7 +11,7 @@
   <name>eXo PLF:: Notes Service</name>
 
   <properties>
-    <exo.test.coverage.ratio>0.44</exo.test.coverage.ratio>
+    <exo.test.coverage.ratio>0.42</exo.test.coverage.ratio>
   </properties>
```

Comment:
please cleanup the dependencies of this artifact to keep only used ones, such as https://github.com/Meeds-io/notes/blob/7d4362d9ce5cc57336b77bed1b8c3813f5dfd263/notes-service/pom.xml#L84-L105

Reply:
Done